src/project.test.ts

```diff
@@ -120,4 +130,147 @@ describe('project', () => {
       });
     });
   });
+  describe('resetUnreleasedPackagesChangelog', () => {
+    it('should reset changelog for packages with changes not included in release', async () => {
+      const project = buildMockProject({
+        rootPackage: buildMockPackage('monorepo'),
+        workspacePackages: {
+          a: buildMockPackage('a', {
+            hasChangesSinceLatestRelease: true,
+          }),
+          b: buildMockPackage('b', {
+            hasChangesSinceLatestRelease: false,
```

Comment:
If you wanted to test that packages that don't have changes don't get restored, it would make sense to set `false` here, but that doesn't seem to be what this test is about, so this could be potentially misleading. Thoughts on setting this to `true` and testing that "b" gets reset along with "c"?